app/ledger.js

```diff
@@ -320,12 +320,13 @@ if (ipc) {
     const win = electron.BrowserWindow.getFocusedWindow()
     if (message === addFundsMessage) {
       appActions.hideMessageBox(message)
+      // See showNotificationAddFunds() for buttons.
+      // buttonIndex === 1 is "Later"; the timestamp until which to delay is set
+      // in showNotificationAddFunds() when triggering this notification.
       if (buttonIndex === 0) {
```

Comment:
These indexes (indices?) would be great as constants, IMO. Instead of checking against 0, it could look like:
`if (buttonIndex === ledgerButton.Later) {`